tasks/main.yml

```diff
@@ -389,3 +389,11 @@
   yum:
     name: "{{ item.name }}"
   with_items: "{{ yum_packages }}"
+
```

Comment:
Please also add a sha256sum check like in L224. You can find the sha256 sum of the latest drone build here: http://downloads.drone.io/release/linux/amd64/drone.sha256